src/krb5_plugin/passkey/passkey.h

```diff
@@ -34,6 +34,9 @@
 #define SSSD_PASSKEY_QUESTION "passkey"
 #define SSSD_PASSKEY_PREFIX "passkey "
 #define SSSD_PASSKEY_REPLY_STATE "ipa_otpd state"
+#define SSSD_PASSKEY_PROMPT "Insert your passkey device, then press ENTER"
+#define SSSD_PASSKEY_PIN_PROMPT "Enter PIN"
```

Comment:
I wonder if we should enable the tuning of these prompts from the sssd configuration.

Reply:
But, IIUC, this is about krb5 plugin used by libkrb5 used by `kinit` directly (or did I get this wrong?)
Not only SSSD has little to do in this use case, but also reading any configs might be undesirable in a plugin.

Reply:
Yes, you are correct. I don't think we need to over-complicate it here, unless it is asked for.